bin/setup

```diff
@@ -23,8 +23,8 @@ chdir APP_ROOT do
   system('bin/yarn')
 
   puts "\n== Preparing and seeding database =="
-  # NOTE: only loading db:structure on primary, because primary_replica is the same db in dev
-  system "bin/rake db:create db:structure:load:primary db:migrate db:seed"
+  # NOTE: only loading db:schema on primary, because primary_replica is the same db in dev
+  system "bin/rake db:create db:schema:load:primary db:migrate db:seed"
```

Comment:
You need to convert the existing `structure.sql` files to schemas

Reply:
Hi @sethherr, we don’t technically need to use a schema.rb file since the `rails db:schema:{load, dump}` tasks work with either Ruby or SQL (see https://github.com/rails/rails/commit/5488686851978c64346a0c03d2b6541cf2a94879#diff-1ed2907b3b8f148c2533558a77673ffa , the `config.active_record.schema_format` is set to `:sql` in `application.rb`.) If you would prefer that we switch back to using a schema.rb, we can invest time into that.

Reply:
Oh, cool! Good to know.